ignite/datadog_checks/ignite/data/conf.yaml.example

```
@@ -0,0 +1,103 @@
+instances:
+
+    ## @param host - string - required
+    ## Ignite host to connect to.
+    #
+  - host: localhost
+
+    ## @param port - integer - required
+    ## Ignite port to connect to.
+    #
+    port: 49112
+
+    ## @param auth_type - string - optional
+    ## The type of authentication to use.
+    ## The available values are "basic" and "digest".
+    #
+    # auth_type: <AUTH_TYPE>
+
+    ## @param user - string - optional
+    ## Username from the credentials needed to connect to the host.
+    #
+    # user: <USERNAME>
```

```suggestion
    ## @param username - string - optional
    ## Username from the credentials needed to connect to the host.
    #
    # username: <USERNAME>
```
Let's use the standard config option for username

---

Unfortunately this comes from jmxfetch, I don't think we have control over it.